tests/phpunit/tests/test-rest-request.php

```diff
@@ -142,6 +146,34 @@ public function test_should_not_define_language_when_not_sent( $data ) {
 		$this->assertNull( $this->pll_rest->curlang );
 	}
 
+	/**
+	 * @testWith [{"type": "post", "route": "/wp/v2/posts", "requiredField": "title", "fieldValue": "Post"}]
+	 *           [{"type": "term", "route": "/wp/v2/categories", "requiredField": "name", "fieldValue": "Category term"}]
+	 *
+	 * @param array $data {
+	 *    @type string $type          Type of content.
+	 *    @type string $route         Route.
+	 *    @type string $requiredField Required field to create the content.
+	 *    @type string $fieldValue    Value of the required field.
+	 * }
+	 */
+	public function test_should_assign_default_language_when_no_language_sent( $data ) {
+		wp_set_current_user( self::$administrator );
```

Comment:
Just a suggestion to maybe use `wp_set_current_user( 1 )` if we only need it for this test, it's up to you.
Otherwise it's ok for me.

Reply:
🤔 It's clearer for me like that.